ui/qt-lib/resources/stylesheet.qss

```diff
@@ -274,6 +274,14 @@ QDialog QPushButton[dialogButtonRole="dialogAction"] {
     padding-right: 80px;
 }
 
+QDialog QPushButton[dialogButtonRole="gifButton"] {
```

Comment:
Please change text color for disabled secondary button, it's present in zeplin. I forgot to do it for some reason :/